rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
@leio thanks, didn't know of this
This list is used by `rosdep` to emerge the needed packages if you use the manual install and to generate ebuilds for the `ros-overlay`